### ISSUE TITLE
Lapack tests on Windows

### ISSUE THREAD

**Issue by mratsim** [mratsim/Arraymancer#212]
https://github.com/mratsim/Arraymancer/pull/211 introduced LAPACK dependencies but I couldn't make the test work. The dll doesn't get detected.

**Comment by pkscn333** [comment]
I am using google translation.

pacman -S mingw64/mingw-w64-x86_64-openblas
(mingw-w64-x86_64-openblas-0.2.20-2)

Using a DLL installed with pacman also worked on windows.

ex.
nim c --define:blas=openblas --define:lapack=openblas ex04_fizzbuzz_interview_cheatsheet.nim

**Comment by data-man** [comment]
@mratsim 
Yes, supports: [docs/build-environment/#mingw-msys-cygwin](https://www.appveyor.com/docs/build-environment/#mingw-msys-cygwin)
For example: [winutils](https://github.com/jjzhang166/winutils/blob/master/appveyor.yml)
